nxviz/edges.py

```diff
@@ -59,7 +71,13 @@ def draw(
     - `G`: A NetworkX graph.
     - `pos`: A dictionary mapping for x,y coordinates of a node.
     - `lines_func`: One of the line drawing functions from `nxviz.lines`
-    - `color_by`: Categorical or quantitative edge attribute key to color edges by.
+    - `color_by`: Categorical or quantitative edge attribute key to
+        color edges by. There are two special value for this parameter
+        when using directed graphs: "source_node_color" and
+        "target_node_color". It should be noted that if these values are
+        set, `node_color_by` also needs to be set.
+    - `node_color_by`: Node metadata attribute key that has been used to
+        color nodes.
```

Comment:
Minor suggestion to use semantic line breaks.

```suggestion
    - `color_by`: Categorical or quantitative edge attribute key to color edges by.
        There are two special value for this parameter
        when using directed graphs: 
        "source_node_color" and "target_node_color". 
        If these values are set, then `node_color_by` also needs to be set.
    - `node_color_by`: Node metadata attribute key 
        that has been used to color nodes.
```

Reply:
👍